### DISCUSSION TITLE
Auth Still Failing

### DISCUSSION THREAD

**Discussion by linova131** [subdavis/kobo-book-downloader#83] [Q&A] [answered]
Hey! So I know there've been recent fixes to the auth flow. I redownloaded and the most recent code changes to this seem to be present, but I'm still getting the perpetual request hang when I try to either access the book list or add a new user. Is anyone else still having trouble there?

**Comment by subdavis** [comment, marked as answer]
Thanks for the report.  Does it work in https://github.com/TnS-hun/kobo-book-downloader or is this something that changed on the Kobo side again?

**Reply by linova131** [reply]
Alright, I just redownloaded from that repo...I still can't seem to add a user/access the book list. With the CLI and the UI it just seems to hang, but I'm not getting back errors that I can see. If I try to add a user in the UI, in the terminal logs, it prints my creds a few times. The actual process to get a captcha code does work. 
Like I said, I'm not convinced it's a Kobo side thing or if it's my user error. I would feel more certain if other people were reporting issues.

On another note: I did just download the most recent zip file, but the kobodl --version returns 0.5.1. Is that relevant?

**Reply by subdavis** [reply]
https://github.com/subdavis/kobo-book-downloader/releases

You definitely don't have the latest version.   Should be 0.9.2

**Reply by linova131** [reply]
Thanks for confirming that. I had to delete some temp files and reinstall again and that time it installed as 0.9.2. For whatever reason, I would download the most recent source code, install and it would persist as 0.5.1. Seems to be working now! Thank you so much.
